src/Global/atoms/DrawerBottomMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Icon } from '../..';
+
+const Container = styled.div`
+  padding: 16px 10px 14px 23px;
+  display: flex;
+  flex-direction: row;
+  border-top: 1px solid ${({ theme }) => theme.colors.divider};
+  align-items: center;
+  cursor: pointer;
+  svg {
+    margin-top: 7px;
+  }
+`;
+
+const ColumnContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding-left: 14px;
+  align-items: flex-start;
+`;
+
+const Title = styled.div`
+  opacity: 0.76;
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  font-size: 12px;
+  font-weight: 500;
+  letter-spacing: 0.34px;
+  color: ${({ theme }) => theme.typography.global.mainMenu.subheader.color};
+`;
+
+const SelectedValue = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  font-size: 10px;
+  font-weight: 500;
+  letter-spacing: 0.29px;
+  color: ${({ theme }) => theme.typography.global.mainMenu.subheader.color};
+  padding-top: 2px;
+  opacity: 0.5;
+`;
+
+interface IDrawerBottomMenu {
+  icon: string,
+  title: string,
+  selectedValue: string,
```

Comment:
Hi Dipak, 
I realized that selectedValue is a string and is only providing the text to be displayed and not the actual value, it does not provide functionality.

Maybe in this case should be more like a subtitle of the DrawerBottomMenu